FRES-266 : Hook Implementation.

Pallabi-Mondal234 opened the pull request.

Pallabi-Mondal234 committed:
FRES-266: Hook Implementation.

Pallabi-Mondal234 committed:
FRES-266: Add extra line.

Pallabi-Mondal234 committed:
FRES-266: Add extra line.

Pallabi-Mondal234 committed:
FRES-266: Hook Implementation in node.

Pallabi-Mondal234 committed:
FRES-266: Add extra line.

Pallabi-Mondal234 committed:
FRES-266: Add comment.

Pallabi-Mondal234 committed:
FRES-266: Add comment.

ankitdebnath003 commented on web/modules/custom/form_module/form_module.routing.yml:
```diff
@@ -0,0 +1,7 @@
+FormModule.custom_form:
```
Key should be in small and first one should same with the module name.

ankitdebnath003 reviewed (changes requested):
1. Where is the custom permission being used?
2. Coding standard should be maintained. Use PHPCS for this.
3. How to verify if these modules are enabled or not?

ankitdebnath003 commented on web/modules/custom/hook_node/hook_node.module:
```diff
@@ -0,0 +1,111 @@
+  $send = true;
+
+  $mailManager->mail($module, $key, $to, $langcode, $params, NULL, $send);
+}
```
If this is a custom hook, where is the documentation of it?

Pallabi-Mondal234 committed:
FRES-266: Update task and proper comment.

Pallabi-Mondal234 committed:
FRES-266: Update permission.

Pallabi-Mondal234 commented:
> 1. Where is the custom permission being used?
> 
>     2. Coding standard should be maintained. Use PHPCS for this.
> 
>     3. How to verify if these modules are enabled or not?

1. I forgot to add the permission i updated it.
and updated all the things that you mentioned.

Pallabi-Mondal234 committed:
FRES-266: Add proper documentation for custom hook.

Pallabi-Mondal234 committed:
FRES-266: Add core.extension.yml file.

Pallabi-Mondal234 committed:
FRES-266: Update .gitignore file
